lib/wfsc/falco_wfsc_loop.m

```diff
@@ -287,7 +299,29 @@
             out.InormHist_tb.unmod = NaN(Itr, mp.Nsbp);
         end
         hProgress = falco_plot_progress_testbed(hProgress, mp, Itr, out.InormHist_tb, Im_tb, DM1surf, DM2surf);
+
     else
         hProgress = falco_plot_progress(hProgress, mp, Itr, out.InormHist, Im, DM1surf, DM2surf, ImSimOffaxis);
+
+        out.InormHist_tb.total = out.InormHist; 

```

Comment:
@gruane @davidsmarx It looks like this ~20 line code block was copy/pasted more or less from the testbed part to the simulation part, but I don't see how or where `Im_tb` gets used. Is it just meant as a temporary variable for computing things before assigning stuff to `out`?

Reply:
Im_tb gets used inside falco_plot_progress_omc for saving iteration-by-iteration output. E.g., lines 247-255 save coherent/incoherent components and probe output for both testbed and simulation.